src/main/java/net/dries007/tfc/TerraFirmaCraft.java

```diff
@@ -53,7 +53,7 @@
 
 @SuppressWarnings("FieldMayBeFinal")
 @Mod.EventBusSubscriber
-@Mod(modid = MOD_ID, name = TerraFirmaCraft.MOD_NAME, useMetadata = true, guiFactory = Constants.GUI_FACTORY, dependencies = "required:forge@[14.23.5.2816,);after:jei@[4.14.2,);after:crafttweaker@[4.1.11,);after:waila@(1.8.25,)")
+@Mod(modid = MOD_ID, name = TerraFirmaCraft.MOD_NAME, useMetadata = true, guiFactory = Constants.GUI_FACTORY, dependencies = "required:forge@[14.23.5.2816,);after:jei@[4.14.2,);after:crafttweaker@[4.1.11,);after:waila@(1.8.25,);before:gregtech")
```

Comment:
Now this change is awkward. The other "after" dependencies are there because we are indeed soft dependant on waila and crafttweaker. Adding before gregtech implies that you want our registry stuff to happen earlier (possibly because of the metal fluids). If so, this is not the solution since other mods also have the possibility of registering earlier than TFC. And by no means adding each mod out there in this line would be the solution.

Reply:
This was an accident and I am in the progress of reverting, oops